unit/analyses/variable-sensitivity/constant_abstract_value/#merge.cpp#

```diff
@@ -0,0 +1,468 @@
+/*******************************************************************\
+
+ Module: Unit tests for constant_abstract_valuet::merge
```

Comment:
File name? `#merge.cpp#`?  Maybe this shouldn't be in the PR?

Reply:
Oops, sorry.